src/main/java/org/molgenis/utils/HgncFile.java

```diff
@@ -0,0 +1,117 @@
+package org.molgenis.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HgncFile {
```

Comment:
How about splitting in a `HgncGenesParser` class that provides a `HgncGenes` class that is used by the `HgncGeneValidator`? (separation of concerns)

Reply:
Good suggestion. I was struggling very much with splitting my classes. I have had them splitted in all sorts of ways, but all the time I ended up in situations in which at least one class didn't make any sense anymore, but this sounds like a sensible way to split up these classes. Thanks!